packages/fonts/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@namehash/fonts",
+  "version": "0.0.1",
+  "files": [
+    "index.ts",
```

Comment:
This shouldn't be here but instead add the exports for main/module etc. See `nameguard/sdk/package.json` for an example.

Reply:
You'll also need to add scripts etc so turso picks this up during dev.